lib/query_relation.rb

```diff
@@ -130,9 +137,22 @@ def offset_value
     options[:offset]
   end
 
-  # @param val [Array<Sting,Symbol>,String, Symbol]
-  def select(*args)
-    append_hash_arg :select, *args
+  # @param columns [Array<Sting,Symbol>,String, Symbol] columns to bring back
+  def select(*columns)
+    append_hash_arg :select, *columns
+  end
+
+  # @param columns [Array<Sting,Symbol>,String, Symbol] columns to bring back
+  def pluck(*columns)
```

Comment:
What happens if someone calls `.pluck` with no arguments?

Reply:
added the use case. looks like pluck is `map(&:values)`

wondering if we want the same implementation

Reply:
active record:

```ruby
Vm.limit(1).pluck(:id, :name) 
= > [1, "vm"]
Vm.limit(1).select(:id, :name, :description).pluck(:id, :name) 
= > [1, "vm"]
Vm.limit(1).select(:id, :name, :description).pluck
= > [1, "vm", 30 parameters]
```

I feel that last one should have kept the select and had 3 parameters not every parameter from the vm.